### PR TITLE
external_script: fix indexerror exception

### PR DESCRIPTION
**Pull request by lasers**
Incinerates https://github.com/ultrabug/py3status/pull/1083.

### CODE REVIEW

File: py3status/modules/external_script.py [hunk 1]
```diff
@@ -75,10 +75,12 @@ def external_script(self):
             output = e.output or e.error
             self.py3.error(output)
 
-        output_text = output_lines[0]
-
-        if self.strip_output:
-            output_text = output_text.strip()
+        if output_lines:
```
Comment (tobes) [inline py3status/modules/external_script.py]: This variable can be undefined if `CommandError` is raised.

Comment (lasers) [reply py3status/modules/external_script.py]: Nice catch. High :five:. I optimized-pushed.

Comment (tobes) [reply py3status/modules/external_script.py]: I prefer the old PR

Comment (lasers) [reply py3status/modules/external_script.py]: K. Old PR w/ column fix.

**Comment by tobes** [comment]
Nice work

**Pull request merged**